wpilibNewCommands/src/main/java/edu/wpi/first/wpilibj2/command/ParallelCommandGroup.java

```diff
@@ -20,6 +20,8 @@
 public class ParallelCommandGroup extends Command {
   // maps commands in this composition to whether they are still running
   private final Map<Command, Boolean> m_commands = new HashMap<>();
+  // a thin read-only view of m_commands for public API usage
+  private final Map<Command, Boolean> m_readonlyCommands = Collections.unmodifiableMap(m_commands);
```

Comment:
Any reason not to make the field `protected` and skip the getter?

Reply:
Mostly personal preference for me; I don't like subclasses reaching directly into the internals of their parent classes.